public/src/classes/states/SettingsState.js

```
@@ -178,6 +178,14 @@ export class SettingsState extends BaseState {
                 if (window.upgradeSystem) {
                     window.upgradeSystem.resetAllUpgrades();
                 }
+
+                // Reset one-time store purchases
+                if (window.storeSystem && window.storeSystem.resetStore) {
+                    window.storeSystem.resetStore();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)